.insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml

```diff
@@ -0,0 +1,56 @@
+_id: req_591ce65b6fd948d68770be96b6c0346a
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1661870574161
+created: 1661284871451
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/invite"
+name: Invite
+description: >
+  Use this endpoint to invite other users to a review. It takes the following
+  parameters in the body:
+
+
+  * `role_id`: The id of the role to assign to the invitees. The table below lists the roles that can be assigned to an invitee:
+
+
+  | Role ID | Role Name    |
+
+  |---------|--------------|
+
+  | 2       | Collaborator |
+
+  | 3       | Viewer       |
+
+  | 4       | Translator   |
+
+  | 6       | Reviewer     |
+
+
+  * `user_emails`: An array of strings containing the emails of the invitees. Eg: `["user1@example.com", "user2@example.com"]`.
```

Comment:
Is this the JSON equivalent for URL-encoded content type like this one:
<img width="346" alt="image" src="https://user-images.githubusercontent.com/855847/187482833-77cbfebb-804a-48e8-8bf8-adba5fce5da9.png">

Reply:
Wdym? The content-type is set to JSON, so your typical JSON arrays will be understood by Rails. 

![image](https://user-images.githubusercontent.com/9405286/187494707-60d3f462-b8b0-4cf5-8e75-6ec77a08f204.png)

Reply:
OK, it is clear now. I was only comparing it to the web endpoint and the `/customizations` but just realized that one is a GET request and has no JSON body.